src/main/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtil.java

```diff
@@ -408,4 +408,47 @@ else if (sibling.getType() == TokenTypes.SINGLE_LINE_COMMENT) {
                 || BlockCommentPosition.isOnPackage(blockComment));
     }
 
+    /**
+     * Gets the content of inline custom tag.
+     *
+     * @param javadocInlineTag javadoc inline tag node.
+     * @return String consisting of the content of inline custom tag.
+     */
+    public static String getContentOfInlineCustomTag(DetailNode javadocInlineTag) {
+        final DetailNode[] childrenOfJavadocInlineTag = javadocInlineTag.getChildren();
+        DetailNode nodeToBeProcessed = childrenOfJavadocInlineTag[2];
+        final DetailNode endingNode =
+                childrenOfJavadocInlineTag[childrenOfJavadocInlineTag.length - 1];
+        String customTagContent = "";
+
+        while (nodeToBeProcessed != endingNode) {
+            customTagContent = customTagContent
+                    .concat(extractContentInlineCustomTag(nodeToBeProcessed));
+            nodeToBeProcessed = getNextSibling(nodeToBeProcessed);
+        }
+        return customTagContent;
+    }
+
+    /**
+     * Extracts the content of inline custom tag recursively.
+     *
+     * @param node DetailNode
+     * @return Text of the node supplied, concatenated text if that node has children.
+     */
+    private static String extractContentInlineCustomTag(DetailNode node) {
+        String result = "";
+        final DetailNode[] children = node.getChildren();
+        if (children.length == 0) {
+            result = node.getText();
+        }
+        else {
+            for (DetailNode child : children) {
+                if (child.getType() != JavadocTokenTypes.LEADING_ASTERISK) {
+                    result = result.concat(extractContentInlineCustomTag(child));
```

Comment:
please use StringBuilder for such cases, string concatenation in loop is not a good idea
https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java

Reply:
Noted, done.